packages/morph/lib/attr-morph.js

```diff
@@ -6,17 +6,20 @@ function updateProperty(value) {
   this.domHelper.setPropertyStrict(this.element, this.attrName, value);
 }
 
-function updateAttribute(value) {
+function updateAttribute(value, namespace) {
```

Comment:
@mixonic This would lead to `updateAttribute` being used for namespaced attributes on svg-namespaced elements.  Instead, should add `namespace` as a condition on the outer `if`, move the `this.namespace` check to the outer `if`'s then clause, and leave the outer `if`'s else clause with `updateProperty`.  

I would also replace `namespace` with `attrNamespace`, just to be extra explicit, but maybe that's going too far.